low-level/sensors/impl/generic/raid_integrity_data.py

```diff
@@ -418,4 +418,4 @@ def _cleanup(self):
         for file in result_files:
             if os.path.getmtime(os.path.join(path, file)) < (current_time - 24*60*60) :
                 if os.path.isfile(os.path.join(path, file)):
-                    os.remove(os.path.join(path, file))
\ No newline at end of file
+                    os.remove(os.path.join(path, file))
```

Comment:
this file has no changes as such as i see. avoid from patch

Reply:
I have not done that as such. I need to check this.

Reply:
just remove file from commit then

Reply:
Ok.